CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 1.2.0
+
+## Features and improvements
+* Add *-sort-key layout property for circle, fill, line ([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
+* Add map-specific access tokens ([#8364](https://github.com/mapbox/mapbox-gl-js/pull/8364))
+* Make requirements for text offset props more precise ([#8418](https://github.com/mapbox/mapbox-gl-js/pull/8418))
```

Comment:
Could we limit non-standard abbreviations ("props") and try to be more precise about style-spec property names? e.g. for this entry:

> Clarify requirements and restrictions for `text-variable-anchor`, `text-radial-offset`, `text-offset` and `text-anchor` properties in documentation